readme.rst

```diff
@@ -206,7 +206,7 @@ Arguments:
 Options:
     --scope             The config value's scope (default, websites, stores)
     --scope-id          The config value's scope ID
-    --decrypt           Decrypt the config value using local.xml's crypt key
+    --decrypt           Decrypt the config value using crypt key defined in env.php
```

Comment:
If this *additional* grammar change is necessary, please also report it against N98-Magerun 1.